docs/_static/icons.css

```diff
@@ -10,3 +10,36 @@
 .custom-icons {
     font-family: 'Custom Icons' !important;
 }
+
+@font-face {
+    font-family: 'Material Icons';
+    font-style: normal;
+    font-weight: 400;
+    src: url('material-icons.woff') format('woff2');
+}
+
+.material-icons {
+    font-family: 'Material Icons';
+    font-weight: normal;
+    font-style: normal;
+    font-size: 24px;
+    /* Preferred icon size */
+    display: inline-block;
+    line-height: 1;
+    text-transform: none;
+    letter-spacing: normal;
+    word-wrap: normal;
+    white-space: nowrap;
+    direction: ltr;
+
+    /* Support for all WebKit browsers. */
+    -webkit-font-smoothing: antialiased;
+    /* Support for Safari and Chrome. */
+    text-rendering: optimizeLegibility;
+
+    /* Support for Firefox. */
+    -moz-osx-font-smoothing: grayscale;
+
+    /* Support for IE. */
+    font-feature-settings: 'liga';
```

Comment:
```suggestion
```
Google's guide adds this, but IE reached eol in June :^)  <sup>ignoring LTSC because can IE please disappear already</sup>
Sphinx doesn't officially support it either, afaik.

Reply:
https://github.com/DisnakeDev/disnake/pull/804/commits/7c955de5692404da6984a1fcd73c095ca86cf519